mkosi

```diff
@@ -2583,6 +2640,8 @@ def install_grub(args: CommandLineArguments, root: str, loopdev: str, grub: str)
 def install_boot_loader_fedora(args: CommandLineArguments, root: str, loopdev: str) -> None:
     install_grub(args, root, loopdev, "grub2")
 
+def install_boot_loader_centos(args:CommandLineArguments, workspace: str, loopdev: str) -> None:
```

Comment:
Replace `workspace` by `root` here.

Reply:
Fixed, thanks for catching this. Do you think a rebase is necessary to reduce the changes to 1 commit? Originally when I submitted the push request it was incredibly messy, and I am unsure on the norms of PR's

Reply:
So with systemd and mkosi, we generally go for "perfect" commits when merging PRs. The idea is that each commit stands completely on its own and works perfectly fine on its own (it builds correctly, all tests pass, no regressions, ...). That makes it easy to run git bisect later and generally keeps the git history clean.

So yes, a rebase that merges everything in one commit would be great.

Reply:
I think I did it, but would appreciate if you could look it over again to make sure I didn't mess anything up in the process. This has been a learning experience and I appreciate the patience.

Reply:
Seems like `workspace` still needs to be updated to `root` here.

Reply:
Fixed, thanks for catching this.